### ISSUE TITLE
Reduce use of "with range" in acceptance tests

### ISSUE THREAD

**Issue by phil-davis** [owncloud/core#39513]
We have acceptance tests that download just a part of a file, use the `range` parameter in the API request. For example, a step like:
````
Then the downloaded content when downloading file "/Shares/testshare/overwritethis.txt" for user "Alice" with range "bytes=0-6" should be "ownClou"
```

In many cases we actually just want to test that downloading a file works. But due to some history, we used a test step that downloads a range of the file. That is testing 2 things at once - downloading, and support for range downloads. That is not optimal when testing systems where downloads work fine, but range is not yet supported or working.

ToDo:
- identify a small set of test scenarios that cover support of range in download and make sure that the feature file name, feature description, and scenario descriptions clearly say that range is being tested.
- adjust other scenarios so that they download the whole file when the scenario just needs to check for valid file content etc.

**Comment by phil-davis** [comment]
The steps in use are:

PublicWebDavContext.php
```
	 * @When /^the public downloads the last public shared file with range "([^"]*)" using the (old|new) public WebDAV API$/

	 * @When /^the public downloads the last public shared file with range "([^"]*)" and password "([^"]*)" using the (old|new) public WebDAV API$/

	 * @When /^the public downloads file "([^"]*)" from inside the last public shared folder with range "([^"]*)" using the (old|new) public WebDAV API$/

	 * @When /^the public downloads file "([^"]*)" from inside the last public shared folder with password "([^"]*)" with range "([^"]*)" using the (old|new) public WebDAV API$/
```

TransferOwnershipContext.php
```
	 * @Then /^the downloaded content when downloading file "([^"]*)" for user "([^"]*)" with range "([^"]*)" from the last received transfer folder should be "([^"]*)"$/
```

WebDav.php
```
	 * @When /^the user downloads file "([^"]*)" with range "([^"]*)" using the WebDAV API$/

	 * @When /^user "([^"]*)" downloads file "([^"]*)" with range "([^"]*)" using the WebDAV API$/

	 * @Then /^the downloaded content when downloading file "([^"]*)" with range "([^"]*)" should be "([^"]*)"$/

	 * @Then /^the downloaded content when downloading file "([^"]*)" for user "([^"]*)" with range "([^"]*)" should be "([^"]*)"$/
```

**Comment by phil-davis** [comment]
Done in #39514 - there were not so many of these to touch. That will be merged when after release 10.9.0 is "stable".
